### PR TITLE
Ignore Visual Studio Code workspace settings

### PR DESCRIPTION
**Pull request by Alxtrkhv**
# Pull Request

### Update

- [x] Template - Update existing `.gitignore` template

## Details

From my point of view, it's more convenient to ignore vscode workspace settings by default.
And this variant is compatible with vscode .gitignore if you later decide to commit your settings and debug tasks.

### CODE REVIEW

**Comment by joeblau** [comment]
Question, would this work better with a stack where you can take Visual Studio code and combine it with Unity?

**Comment by Alxtrkhv** [comment]
Just to make sure I understood you correctly:
Do you propose combining two gitignores (one for unity and one for vscode)?
If yes: vscode gitignore (https://github.com/toptal/gitignore/blob/master/templates/VisualStudioCode.gitignore) uses different approach. As far as I understand, it ignores extension-related files but doesn't ignore vscode workspace settings. 
The approach I proposed ignores all vscode-related files (entire .vscode folder)

**Comment by brennanfee** [comment]
The current file has this:

```
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
*.code-workspace
```

Which is correct according to their own documentation on what files should be checked in for a team versus which files should be ignored.  The four files explicitly included (with the exclamation point) are necessary for many projects (particular team projects with more than one individual contributing).

Given that the current one already includes the `*.code-workspace` item you are looking to add, I think your use case is covered and so your PR is redundant.

So, in short, the "entire" folder should NOT be ignored but all files EXCEPT those critical 4.

Here is some relevant documentation from Microsoft on the four "special" files.

On settings.json:  https://code.visualstudio.com/docs/getstarted/settings#_settings-file-locations
On luanch.json:  https://code.visualstudio.com/docs/editor/debugging
On tasks.json:  https://code.visualstudio.com/docs/editor/tasks
On extensions.json:  https://code.visualstudio.com/docs/editor/extension-gallery#_workspace-recommended-extensions

**Comment by serverhorror** [comment]
Please don't merge this but use the suggestion from @brennanfee. Ignoring all of `.vscode/` would break a lot of things.
